var/spack/repos/builtin/packages/dbcsr/package.py

```diff
@@ -53,7 +55,8 @@ class Dbcsr(CMakePackage, CudaPackage, ROCmPackage):
 
     depends_on("cmake@3.17:", type="build", when="@2.1:")
     depends_on("cmake@3.10:", type="build")
-    depends_on("py-fypp", type="build")
+    depends_on("cmake"3.22:", type="build", when="@2.6:") # maybe 2.6:?
```

Comment:
I can't add a change suggestion with the deleted line so here's a separate snippet of code to replace the `cmake` `depends_on` clauses:

```
    depends_on("cmake@3.10:", type="build")
    depends_on("cmake@3.12:", type="build", when="@2.1:")
    depends_on("cmake@3.17:", type="build", when="@2.2:")
    depends_on("cmake"3.22:", type="build", when="@2.3:")
```
Taken from the main `CMakeLists.txt` in dbcsr.